aws/dynamodb_test.go

```diff
@@ -81,7 +82,7 @@ func TestGetTablesDynamo(t *testing.T) {
 	)
 
 	require.NoError(t, err)
-	getAllDynamoTables(awsSession, time.Now().Add(1*time.Hour*-1), db)
+	getAllDynamoTables(awsSession, time.Now().Add(1*time.Hour*-1), config.Config{}, db)
```

Comment:
Result of function invocation not checked, I think at least should be checked for errors

Reply:
Fixed